javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,41 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  
+    studentName === "" ? console.log("Please add a name!"):undefined;
+
+    if (studentName == "Queen"  || studentName === "queen"){
+        class07Students.push("Queen");
+        return;
+    } else if(class07Students.length >= 6 && studentName !== "Queen"){
+        return "Sorry there is no more space for other students.";
+    }  else if (class07Students.length < 6 && !class07Students.includes(studentName) ) {
```

Comment:
same as before, you don't need this part of the statement "class07Students.length < 6 "